test/integration/drain_close_integration_test.cc

```diff
@@ -0,0 +1,89 @@
+#include "test/integration/http_protocol_integration.h"
+
+namespace Envoy {
+namespace {
+
+using DrainCloseIntegrationTest = HttpProtocolIntegrationTest;
+
+// Add a health check filter and verify correct behavior when draining.
+TEST_P(DrainCloseIntegrationTest, DrainClose) {
```

Comment:
I assume these should be removed from protocol integration test?

Reply:
Ah oops thought I did that.